pages.fr/linux/apache2ctl.md

```diff
@@ -2,7 +2,7 @@
 
 > L'outil d'Interface en Lignes de Commandes (ILC) pour administrer le serveur web HTTP Apache.
 > Cette commande est disponible sur une distribution Debian. Pour les distributions basées Red Hat, voir `httpd`.
-> Plus d'information : <https://manpages.debian.org/latest/apache2/apache2ctl.8.en.html>.
+> Plus d'informations : <https://manpages.debian.org/latest/apache2/apache2ctl.8.en.html>.
```

Comment:
```suggestion
> Plus d'informations : <https://manpages.debian.org/latest/apache2/apache2ctl>.
```

Reply:
I suggest here as in that other issue @navarroaxel that we make minimal adjustments to the URL - i.e. removal of the `.en`, but leaving the `.html` alone since that's what it redirects to.

Reply:
Should we do that in this PR or in a seperate one fixing all `manpages.debian.org` links?

Reply:
I'd prefer a separate PR.